### PR TITLE
Contributing guidelines

### LINKED ISSUE #191: Document private data policy on issues

### PR DESCRIPTION
**Pull request by faloi**
Closes #191

### CODE REVIEW

**Comment by faloi** [comment]
@flbulgarelli anything else you want to say? If it's ok, I'll continue with the English version.

**Comment by flbulgarelli** [comment]
Me parece correctín!

Digo...

It looks good to me.  We should add this to all main repos. 

I'd add however notes about the following: 
- Language: we should clearly state that issues can be submitted both in spanish and english. And that issues added to guides should be in the language of the guide.  
- That we adhere to [the contributor covenant](http://contributor-covenant.org/) which is quite standard nowadays
- We may also add guidelines about contributing to the source code. Now I am short of ideas. @fedescarpa? @aguspina ?

**Comment by faloi** [comment]
> Language: we should clearly state that issues can be submitted both in spanish and english. And that issues added to guides should be in the language of the guide.

Ok with this one, the others are not suitable for this repository because it has no code.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**IMPORTANTE:** podés cargar tus sugerencias en español o en inglés, ¡usá el idioma en el que te puedas expresar mejor!
+
+**IMPORTANT:** you can upload issues both in english and spanish, choose the language that better helps you to express yourself!
+
+:warning: :warning: :warning:
+
+# ![Argentina](http://mumuki.io/static/argentina.png) Español
+
+¡Hola! Antes que nada, queremos agradecerte por la contribución que estás por hace; tener una comunidad proponiendo cambios y mejoras es la única forma de que la plataforma mejore. Para entendernos mejor, te pedimos por favor que prestes atención a los siguientes puntos:
```
Comment (flbulgarelli) [inline CONTRIBUTING.md]: hacer

Comment (flbulgarelli) [inline CONTRIBUTING.md]: I'd add some emojis to keep this contribution guidelines informal and following mumuki wording style

**Pull request merged**